.github/workflows/conformance-runtime.yaml

```diff
@@ -358,3 +367,9 @@ jobs:
           path: |
             cilium-junits/*.xml
           retention-days: 2
+
+      - name: Publish Test Results As GitHub Summary
+        if: ${{ always() }}
+        uses: aanm/junit2md@86789108ea0fd9be947a4232dd560e2fa3b9467a # v0.0.1
```

Comment:
Should we move that to the Cilium organization?

Reply:
Yes, once we address https://github.com/cilium/community/issues/27